src/Draco.Compiler/Internal/Symbols/Synthetized/PrimitiveTypeSymbol.cs

```diff
@@ -10,12 +10,22 @@ internal class PrimitiveTypeSymbol : TypeSymbol
 {
     public override Symbol? ContainingSymbol => null;
     public override string Name { get; }
+    public override string MetadataFullName { get; }
+    public override string MetadataName => this.MetadataFullName.Split('.')[^1];
```

Comment:
Are these needed for "just primitives"? Since we can only have docs for metadata-backed primitive symbols (and you covered that), no?